routing.md

```diff
@@ -13,6 +13,7 @@
     - [Namespaces](#route-group-namespaces)
     - [Sub-Domain Routing](#route-group-sub-domain-routing)
     - [Route Prefixes](#route-group-prefixes)
```

Comment:
Would it be better to rename **Route Prefixes** to **Route URi Prefixes** for a bit more clarity?